src/types/config.js

```diff
@@ -54,13 +53,33 @@
  */
 
 /**
- * @typedef RequestConfigFromFrameOptions
+ * May be provided by `AnnotatorConfigFromHost` to configure a known ancestor
+ * frame as the "embedder" frame. `ConfigFromHost` will be requested from this
+ * frame, and additional messaging (via JSON RPC) may be configured between
+ * the sidebar frame and this embedder frame.
+ *
+ * @typedef EmbedderFrameConfig
  * @prop {number} ancestorLevel
  * @prop {string} origin
  */
 
 /**
- * Configuration set by the embedder of the client and used by the sidebar.
+ * An "embedder frame" may provide configuration to be notified (via JSON RPC)
+ * of qualifying annotation activity from the sidebar frame.
```

Comment:
This functionality is currently dependent on settings in `requestConfigFromFrame`, it will only work in cases involving an "embedder" frame. There may absolutely be cases in the future where we'd want to be able to communicate annotation activity between the sidebar and annotator/host frames (i.e. in a "normal", non-LMS setup) but at present that's not possible so I'm explicitly only allowing this configuration on "embedder"-provided host configuration.